### ISSUE TITLE
weight initialization

### ISSUE THREAD

**Issue by evagian** [crisb-DUT/DnCNN-tensorflow#34]
Hello :)
Could you please help me with something?... 
It is not clear to me in which part of your code you initialize the weights of the CNN.

**Comment by anlijuncn** [comment]
It's also confused to me too. I recommend you to read https://github.com/SaoYan/DnCNN-PyTorch
I don't know if we need some skills to initialize these weights with BN layers. If you have reached to the final result in paper. Please SHARE on you github!! I am now working on it, but only could get PSNR 29.14 for DnCNN-S. :)

**Comment by evagian** [comment]
Thank you @anlijuncn 

I think that they are using the default tf.layers.conv2d initializer which is Xavier but I am not sure about that.

**Comment by anlijuncn** [comment]
Yes, in crisb-dut's code he is using default initializer. I don't know whether methods could improve the result. The author ZhangKai has released Pytorch and Keras version code on github/cszn.

**Comment by eastchun** [comment]
Default kernel initializer for tf.layer.conv2d is "glorot_uniform_initializer".

Please refer to code lines '315 - 317' in : https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/variable_scope.py
